Jenkinsfile

```
@@ -92,7 +92,7 @@ pipeline {
         }
 
         stage('Elastic Agent x-pack Windows'){
-          agent { label 'windows-immutable && windows-2019' }
+          agent { label 'windows-immutable && windows-2012' }
```

```suggestion
          agent { label 'windows-immutable && windows-2012-r2' }
```

And in the other sections